packages/storybook-test/src/i18n.ts

```diff
@@ -0,0 +1,10 @@
+import i18n from 'i18next';
+import HttpBackend from 'i18next-http-backend';
+import { initReactI18next } from 'react-i18next';
+
+i18n.use(HttpBackend).use(initReactI18next).init({
+  debug: true,
+  fallbackLng: 'nl',
```

Comment:
Is het ook mogelijk om een error te krijgen in plaats van een fallback te hebben?

Reply:
Wat bedoel je met een error? `console.log`, `throw`? Of een bepaalde fouttekst?